mkdocs/docs/tutorials/deployments/slo-validation-chaos.md

```diff
@@ -77,23 +79,26 @@ Due to chaos injection, and the fact that the number of replicas of the app in t
 iter8ctl assert -c winnerFound
```

Comment:
why do we not use the `-c completed` assertion as well? I noticed it in the first experiment and would expect it to be the standard query.

Reply:
Fixed.